### PR TITLE
Postman fix

### PR DESCRIPTION
**Pull request by TayGov**
# Description

This PR includes the following proposed change(s):

Update postman collection

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [ ] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/backend/Dockerfile.efiling-api [hunk 1]
```diff
@@ -32,12 +32,12 @@ RUN git checkout $STARTERS_V  && \
...
 WORKDIR /skywalking
 
-RUN wget https://archive.apache.org/dist/skywalking/$SW_V/apache-skywalking-apm-$SW_V.tar.gz && \
+RUN wget https://mirror.csclub.uwaterloo.ca/apache/skywalking/$SW_V/apache-skywalking-apm-$SW_V.tar.gz && \
     tar -xzvf apache-skywalking-apm-$SW_V.tar.gz && \
     rm apache-skywalking-apm-$SW_V.tar.gz
 
 # add spring boot plugin
-RUN mv apache-skywalking-apm-bin/agent/optional-plugins/apm-spring-annotation-plugin-8.5.0.jar apache-skywalking-apm-bin/agent/plugins/apm-spring-annotation-plugin-8.5.0.jar
+RUN mv apache-skywalking-apm-bin/agent/optional-plugins/apm-spring-annotation-plugin-8.5.0.jar apache-skywalking-apm-bin/agent/plugins/apm-spring-annotation-plugin-8.4.0.jar
```
Comment (karthikmsqa) [inline src/backend/Dockerfile.efiling-api:41]: Nice.. thought we can't get a older version from the mirror

File: src/backend/Dockerfile.efiling-reviewer-api [hunk 2]
```diff
@@ -7,7 +7,7 @@ ARG SERVICE_NAME=efiling-reviewer-api
 ARG SKIP_TESTS=false
 ARG MVN_PROFILE=default
 ARG STARTERS_V
-ARG SW_V=8.5.0
+ARG SW_V=8.4.0
```
Comment (karthikmsqa) [inline src/backend/Dockerfile.efiling-reviewer-api:10]: Same thing abt version no.

File: src/backend/Dockerfile.efiling-api (hunk 1 above)
Comment (TayGov) [reply src/backend/Dockerfile.efiling-api:41]: Good catch.  I updated on my other pr. Shouldn't affect this one. But that one need to be merged first

**Review by karthikmsqa — APPROVED** [review]
Awesome!

**Pull request merged**